### PR TITLE
AM Dashboard: Develop Shell Script

### PR DESCRIPTION
**Pull request by ssivas6**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/SDT-209

### Change description ###

Shell script to generate and send the stats file to the AM Dashboard

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: scripts/amdashboard/sdt/amdashboard-sdt-stats.sh [hunk 1]
```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+AZURE_DB="civil_sdt"
+CURRENT_DATE=$(date +%d"/"%m"/"%Y)
+CURRENT_TIME=$(date +%T)
```
Comment (paulridout) [inline scripts/amdashboard/sdt/amdashboard-sdt-stats.sh:4]: Could you add a blank line after the shebang on line 1 and move the AZURE_DB declaration after the CURRENT_DATE and CURRENT_TIME?  The date and time mark when the script was run, so it makes sense to have them as early on in the script as possible.

File: scripts/amdashboard/sdt/amdashboard-sdt-stats.sh [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+OUTPUT_DIR="/tmp"
+OUTPUT_FILENAME="SDTCLOUD_stats.txt"
+OUTPUT_FILEPATH=${OUTPUT_DIR}/${OUTPUT_FILENAME}
+ID_FILENAME=bais_id
+ID_FILEPATH=${OUTPUT_DIR}/${ID_FILENAME}
+QUERY_RECENT_SUBMISSIONS="./queries/amdashboard-sdt-recent.sh"
```
Comment (paulridout) [inline scripts/amdashboard/sdt/amdashboard-sdt-stats.sh:10]: Add a blank line before this declaration.  This is just a cosmetic change to group related variables together.

**Comment by paulridout** [comment]
Script will be invoked by an Azure pipeline.  The file to configure the pipeline will be added in a separate pull request.

**Review by cpareek — COMMENTED** [review]
Have we able to test this scripts to make sure its executing as expected?

**Comment by paulridout** [comment]
> Have we able to test this scripts to make sure its executing as expected?

Script has been successfully tested locally using a Docker hosted SFTP server.  We've added some screen shots showing this and a copy of the file produced to the Jira ticket.

**Review by cpareek — APPROVED** [review]

**Pull request merged**